server/routes/creatingLicences/types/additionalConditions.ts

```diff
@@ -0,0 +1,16 @@
+import { Expose, Type } from 'class-transformer'
+import { ValidateIf, ValidateNested } from 'class-validator'
+import RegionOfResidence from './regionOfResidence'
+
+class AdditionalConditions {
+  @Expose()
+  additionalConditions: string[]
+
+  @Expose()
```

Comment:
So this will grow with validation objects of different types, one per condition, and each with their own rules? Nice.

Reply:
Yes it will, but I anticipate that there might be some change around here regarding how these classes look depending on how the new design looks